### ISSUE TITLE
`SASS_STYLE_COMPRESSED` fails to eliminate empty rules with comments

### ISSUE THREAD

**Issue by asottile** [sass/libsass#894]
Input:

```scss
a {/**/}
b {content: 'something so I have a non-empty expected output'}
```

Expected output (ruby sass + pyscss):

```
b{content:'something so I have a non-empty expected output'}
```

Output from libsass (using HEAD of https://github.com/dahlia/libsass-python (libsass 3.1.0))
```
a{}b{content:'something so I have a non-empty expected output'}
```

**Comment by xzyfer** [comment]
Thanks for the report @asottile . I can confirm this is still present on master.

Could you please open  sass-spec PR? Here's a good example to get you started https://github.com/sass/sass-spec/pull/260

**Comment by asottile** [comment]
@xzyfer I didn't see an obvious way to add a regression test for compressed mode, could you give me a pointer or two?

**Comment by xzyfer** [comment]
@asottile sorry I was under the mistaken impression `a` would be removed in non-compressed modes as well. You're correct we don't support compressed for specs. 

There is currently work happening to better deal with output modes so hopefully this will be fixed as part of that. It's hard to say when that'll land though since these minor stylistic issues tend to be a low priority. 

***

As an aside in my opinion compressed mode is best avoided (even in ruby sass) in favour proper css minifiers like http://cssshrink.com/. The compressed mode output is rather naive and leaves a lot on the table.

**Comment by asottile** [comment]
Yeah that's fine, I'm not too concerned about this (we've identified as not a blocker from moving from pyscss to libsass), just figured I'd be a good sumaritan and report :)

**Comment by xzyfer** [comment]
> just figured I'd be a good sumaritan and report

Thanks <3 !

**Comment by mgreter** [comment]
This should be solved by #910!
